pkgs/development/python-modules/cfclient/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, cflib
+, appdirs
+, pyzmq
+, pyqtgraph
+, pyyaml
+, numpy
+, vispy
+, pyserial
+, pyqt6
+, pyqt6-sip
+, qasync
+, packaging
+, qtm
+, xorg
+, wrapQtAppsHook
+, pythonRelaxDepsHook
+, qt5
+}:
+
+buildPythonPackage rec {
+  pname = "cfclient";
+  version = "2023.6";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "bitcraze";
+    repo = "crazyflie-clients-python";
+    rev = "${version}";
+    hash = "sha256-QQuULICE/RjhZj3GGeFJ0hFrFJJ9FphQndeqWQCf59A=";
+  };
+
+  postPatch = ''
+    echo '{"version": "${version}"}' > src/cfclient/version.json
+  '';
+
+  propagatedBuildInputs = [
+    cflib
+    appdirs
+    pyzmq
+    pyqtgraph
+    pyyaml
+    numpy
+    vispy
+    pyserial
+    pyqt6
+    pyqt6-sip
+    qasync
+    packaging
+    qtm
+    qt5.qtbase
+    xorg.libXinerama
```

Comment:
Libraries that are linked against, probably qtbase, libXinerama, should go to buildInputs.